client/src/themes/theme.js

```diff
@@ -200,6 +201,31 @@ export const useStyles = makeStyles((theme) => ({
   textMarginLeft: {
     marginLeft: '10px'
   },
+  questionAnswerButtonContainer: {
```

Comment:
It feels like you can merge answerContainer and QAButtonContainer or do some kind of inherit?